doc/local-run.md

```diff
@@ -2,7 +2,7 @@
 
 Everything you need to do is to create the release. To achieve that run:
 `make rel`. Now you are ready to test our scenario locally with one Amoc node;
-to start the node run `_build/default/rel/amoc/bin/amoc console`.
+to start the node run `_build/default/prod/rel/amoc/bin/amoc console`.
```

Comment:
The path is almost correct. I've just made a clean build (without anything in `_build` directory) and the correct path to amoc binary is:
```suggestion
to start the node run `_build/prod/rel/amoc/bin/amoc console`.
```